package.json

```
@@ -1,9 +1,9 @@
 {
   "name": "vscode-language-fsh",
   "displayName": "FHIR Shorthand",
-  "description": "FHIR Shorthand (FSH) Language Support by MITRE",
+  "description": "FHIR Shorthand (FSH) Language Support by HL7 community",
   "version": "1.16.1",
-  "author": "The MITRE Corporation",
+  "author": "HL7 community",
```

Here I'd also just refer to HL7 as the author:
```suggestion
  "author": "Health Level Seven International",
```